clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -250,7 +255,7 @@ public PrototypeAsyncConsumer(final Time time,
             // no coordinator will be constructed for the default (null) group id
             if (!groupId.isPresent()) {
                 config.ignore(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG);
-                //config.ignore(ConsumerConfig.THROW_ON_FETCH_STABLE_OFFSET_UNSUPPORTED);
+                config.ignore(ConsumerUtils.THROW_ON_FETCH_STABLE_OFFSET_UNSUPPORTED);
```

Comment:
Moving the package-level variable and method from `ConsumerConfig` also fixes this issue.

Reply:
I've moved the core variable back to `ConsumerConfig` but left a copy in `ConsumerUtils` with an explanation of why it's redefined there.